src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
+  padding: 0;
+
+  --hilight-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: flex;
+}
+
+img {
+  width: 40px;
```

Comment:
Avoid using tag names for styling (except `html` and `body`). This can lead to unexpected results if new images are added to the page. It would be better to add a class to the img tag and style it. (Checklist: Don't use tag names for styling)